protocol/src/main/kotlin/models/Close.kt

```diff
@@ -0,0 +1,24 @@
+package models
+
+import typeid.TypeID
```

Comment:
Just as an aside, when we first introduced the [`typeid-js` dependency](https://github.com/jetpack-io/typeid-js) in `tbdex-js` we married ourselves to that particular package, which may or may not have multi-language support. In this case it appears it does have multi-language support. But, just a forewarning on rubber stamping new dependencies.

Reply:
I would suggest NOT rolling our own token lib and implementing typeid in languages where it's currently not supported.